test-fixtures/utils/03/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "TOS 2024-07-30T22:00:00+03:00 > NOW 2024-07-30T22:01:00+03:00 < TOE 2024-07-31T01:00:00+03:00",
+  "only": false,
+  "method": "isOfflinePeriod",
+  "importOfflinePeriod": {
+    "startHour": 22,
+    "lengthHours": 3
+  },
+  "nowTime": "2024-07-30T19:01:00.000Z",
+  "expectedResult": true
```

Comment:
If offline period starts at 22:00 and lasts for 3 hours, should the isOfflinePeriod at 19:01 return false?

Reply:
Timezoning. That timestamp is UTC so in our containers Date adds 3 hours to it